scripts/deployers/rm-slashingManager/generatePayload.s.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import {Script, stdJson, console} from "lib/forge-std/src/Script.sol";
+
+// These imports get generated by npm run generate:interfaces
+import {Governance} from "../../helpers/interfaces/Governance.generated.sol";
+import {Registry} from "../../helpers/interfaces/Registry.generated.sol";
+
+contract DeployFix is Script {
+    using stdJson for string;
+
+    Registry registry = Registry(0x33a02E6cC863D393d6Bf231B697b82F6e499cA71);
+
+    bytes32 private constant SLASHING_MANAGER = keccak256("slashingManager");
+
+    address governance = 0x6e7a5820baD6cebA8Ef5ea69c0C92EbbDAc9CE48;
+
+    address gSafeAddress = 0xCaf0aa768A3AE1297DF20072419Db8Bb8b5C8cEf;
```

Comment:
Just minor feedback: We can change the name to timelock, it would be more accurate.

Reply:
you're right. Copy paste :D